### PR TITLE
Scope to sidebar panel the navbar dropdowns positioning on mobile

### PR DESCRIPTION
**Pull request by nabaroa**
Fixed two issues:

1.- Dropdown menu on max-medium screen sizes were behaving like sidebar dropdown menus even they were not part of the sidebar. 

This was the reason of this bug:

**before**
image

**now**
image

2.- For the same reason we were losing the arrow on this screen sizes.
image

### CODE REVIEW

File: dist/css/bootstrap.css [hunk 1]
```diff
@@ -7948,6 +7934,20 @@ select[multiple].input-group-sm > .input-group-addon {
 .navbar-nav > li > .dropdown-menu {
   margin-top: 56px;
 }
+.navbar-nav > li > .dropdown-menu:before {
```
Comment (StevePole) [inline dist/css/bootstrap.css:7937]: Why :before?

File: docs/dist/css/bootstrap.css [hunk 2]
```diff
@@ -8371,6 +8350,23 @@ fieldset[disabled] .navbar-inverse .btn-link:focus {
          -o-transform: translateX(-264px);
             transform: translateX(-264px);
   }
+  .navbar-collapse .navbar-collapse-panel .dropdown-menu {
```
Comment (StevePole) [inline docs/dist/css/bootstrap.css]: Don't think you need -panel in here

File: dist/css/bootstrap.css (hunk 1 above)
Comment (nabaroa) [reply dist/css/bootstrap.css:7937]: The `:before` is used to show the little arrow of the dropdown.
image

**Review by StevePole — COMMENTED** [review]
It's a bit hard to understand what these changes are for.  I am a little concerned that there's a misunderstanding of the roles of these classes.

navbar-nav - styles a list to be a nav list.
navbar-collapse - container for the section of the navbar that disappears on mobile.

A dropdown-menu inside a navbar-collapse would be displayed differently on mobile than on not inside the navbar collaps (which retains the same styling at all widths).

If you already got that, that's good :-)

**Comment by nabaroa** [comment]
Yes, the main purpose of this PR is to create the difference that you comment: to scope the dropdowns when they are inside the `.navbar-collapse`. 
Currenctly all the dropdowns for the small mediaquery are behaving as if they were inside the `.navbar-collapse`. This breaks all the dropdowns that we want to keep outside the sidebar. 
For instance:

image

With this change, a dropdown that is not inside the `navbar-collapse` is going to maintain its original styles.

**Comment by nabaroa** [comment]
Demo: https://new-grasshopper-73.localtunnel.me/examples/nested-navbar/

**Comment by nabaroa** [comment]
Demo: https://itchy-dingo-15.localtunnel.me/examples/nested-navbar/

**Review by StevePole — APPROVED** [review]
Ok 👍

**Pull request merged**